eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxInfo.hs

```diff
@@ -95,13 +95,21 @@ data TranslationError
   | ByronOutputInContext
   | TranslationLogicErrorInput
   | TranslationLogicErrorRedeemer
+  | TranslationLogicErrorDoubleDatum
+  | LanguageNotSupported
+  | InlineDatumsNotSupported
+  | ReferenceScriptsNotSupported
```

Comment:
Note that there is no worry of breaking predicate failure serialization here in the node-to-client protocol since these errors are never raised in the Alonzo era.